### PR TITLE
chore: update Camunda Modeler screenshots to v5.15

### PR DESCRIPTION
**Pull request by philippfromme**
## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [x] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by philippfromme** [comment]
Related to https://github.com/camunda/camunda-modeler/issues/3775.

File: docs/components/modeler/desktop-modeler/img/deploy-diagram-camunda-cloud-success.png
Comment (barmac) [inline docs/components/modeler/desktop-modeler/img/deploy-diagram-camunda-cloud-success.png:1]: Something went wrong here. Is the client/cluster gone?

File: docs/components/modeler/desktop-modeler/img/start-instance-successful.png
Comment (barmac) [inline docs/components/modeler/desktop-modeler/img/start-instance-successful.png:1]: Same as in the previous comment.

File: docs/self-managed/modeler/desktop-modeler/img/deploy-success.png
Comment (barmac) [inline docs/self-managed/modeler/desktop-modeler/img/deploy-success.png:1]: Something's wrong here too.

**Review by barmac — CHANGES_REQUESTED** [review]
Some of the screenshots need to be regenerated with correct credentials.

**Review by akeller — CHANGES_REQUESTED** [review]
The footer shows a number of different platform values:

* Camunda Platform 8.1
* Camunda Platform 8.0
* Camunda Platform 8 (Zeebe 1.3)
* Camunda Platform 8.2

This could create the illusion or perception that these screenshots are not up-to-date. Can this be adjusted?

If this is a big lift to make these changes, it would be good to remove unsupported or soon to be unsupported versions like Zeebe 1.3 and Camunda Platform 8.0.

BTW, we are actively dropping "Platform" in the docs to align with [this effort](https://camunda.slack.com/archives/C05P016GR5M/p1692916003827649).

**Comment by barmac** [comment]
> The footer shows a number of different platform values:
> 
> * Camunda Platform 8.1
> * Camunda Platform 8.0
> * Camunda Platform 8 (Zeebe 1.3)
> * Camunda Platform 8.2
> 
> This could create the illusion or perception that these screenshots are not up-to-date. Can this be adjusted?

This can be done when we implement https://github.com/camunda/camunda-modeler/issues/3841. We can generate new screenshots, and adjust it afterwards.

File: docs/components/modeler/desktop-modeler/img/deploy-diagram-camunda-cloud-success.png
Comment (philippfromme) [reply docs/components/modeler/desktop-modeler/img/deploy-diagram-camunda-cloud-success.png:1]: I'll look into that.

**Comment by philippfromme** [comment]
> Some of the screenshots need to be regenerated with correct credentials.

I resolved the issue by updating the secrets to point to a running cluster. Will likely have to be updated again soon.

**Review by barmac — APPROVED** [review]

**Review by akeller — APPROVED** [review]

**Pull request merged**